man/GDS.Rd

```diff
@@ -0,0 +1,45 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
You need to regenerate the Rd doc files by running devtools::check() in RStudio, as they still contain references to `short_version` and `configuration_filepath`.

Reply:
ok, done.